docs/operation/changing_control_plane_ingress_ip.rst

```diff
@@ -1,8 +1,16 @@
 Changing the Control Plane Ingress IP
 =====================================
 
-#. On the bootstrap node, update the ``ingressIP`` field from
-   ``networks.controlPlane`` in the Bootstrap configuration file.
+This procedure allow to change the Control Plane Ingress IP and also to move
+from MetalLB configuration to non-MetalLB configuration and the opposite.
+
+.. note::
+
+  If you switch from MetalLB to non-MetalLB this procedure do **not** remove
+  MetalLB installation, it just do not use it for MetalK8s components
```

Comment:
It's debatable btw (I may want to update the design with this :thinking:)

Reply:
Fine by me. Let's wait for someone to report this as an issue before we do anything.